admin/app/views/workarea/admin/pricing_skus/new.html.haml

```diff
@@ -34,6 +34,8 @@
           %span.property__name= t('workarea.admin.fields.discountable')
           = toggle_button_for 'sku[discountable]', @sku.discountable?, title_true: t('workarea.admin.pricing_skus.allow_discounting'), title_false: t('workarea.admin.pricing_skus.disallow_discounting')
 
+        = append_partials('admin.add_cost', sku: @sku)
```

Comment:
Looks like `admin.pricing_sku_fields` is already here too!

Reply:
I do agree with using the existing append point for the new form as it will look fine with the cost underneath the MSRP.  This also depends on the edit form as I reference in the edit form above.